apps/api-evidence/src/app/utils/deps.py

```diff
@@ -11,7 +11,7 @@
 from app.db.session import SessionLocal
 from app.storage import Encryption, Storage, StorageBackend
 
-token_auth = OAuth2PasswordBearer(tokenUrl="/auth/email")
+token_auth = OAuth2PasswordBearer(tokenUrl="/api/auth/email")
```

Comment:
The black formatter should convert this to single quotes, right? I guess the CI is not set up properly to check this. I find that running `pnpx nx format api-evidence` doesn't run the formatter, but `pnpx nx api-evidence:format` does. I don't see an easy way for the CI command to run correctly (`pnpx nx format:check --base=origin/main`), but I will look in to it.

Reply:
I tried a bit on nx, it seems the correct command for that would be `pnpx nx affected:format --base=origin/main` for formatting the affected projects.

Reply:
I think that works, but the goal is to check the format and fail if there is any change. Maybe there is a standard parameter like `--check`, and the python plugin needs to implement it.

Reply:
or probably a general way to pass arguments to the executor of the python plugin